tasks.schema.json

```diff
@@ -429,6 +435,9 @@
       "additionalProperties": false,
       "description": "Execution options for this task (on Post-Job stage)",
       "properties": {
+        "Node20": {
+          "$ref": "#/definitions/executionObject"
+        },
         "Node16": {
           "$ref": "#/definitions/executionObject"

```

Comment:
[not related to this line] releases/5.x is used for other changes - we'll need to re-target these changes